scans/worker.py

```diff
@@ -0,0 +1,80 @@
+import os
+import json
+
+from scans import decimalencoder
+import boto3
+dynamodb = boto3.resource('dynamodb')
+from scans import Response
+
+
+def worker(event, context):
+    table = dynamodb.Table(os.environ['DYNAMODB_TABLE'])
+
+    # Example event to play with
+    # {
+    #     "Records": [
+    #         {
+    #             "eventID": "1",
+    #             "eventVersion": "1.0",
+    #             "dynamodb": {
+    #                 "Keys": {
+    #                     "id": {
+    #                         "S": "a88cb45c-0ac1-47b3-865b-a92f05d7612b"
+    #                     }
+    #                 },
+    #                 "NewImage": {
+    #                     "target": {
+    #                         "S": "ssh.mozilla.com"
+    #                     },
+    #                     "port": {
+    #                         "I": 22
+    #                     },
+    #                     "id": {
+    #                         "S": "a88cb45c-0ac1-47b3-865b-a92f05d7612b"
+    #                     }
+    #                 },
+    #                 "StreamViewType": "NEW_IMAGE",
+    #                 "SequenceNumber": "111",
+    #                 "SizeBytes": 128
+    #             },
+    #             "awsRegion": "us-west-2",
+    #             "eventName": "INSERT",
+    #             "eventSourceARN": "event:source:ARN",
+    #             "eventSource": "aws:dynamodb"
+    #         }
+    #     ]
+    # }
+
+    for record in event["Records"]:
+        if not record["eventName"] == "INSERT":
```

Comment:
I saw some examples that distinguish INSERT from MODIFY, so I expect that the modify/update operation below will not send this into an infinite stream handler loop.